### ISSUE TITLE
Task List Broken. Completing Single Tasks From Home Page Broken

### ISSUE THREAD

**Issue by gdginc** [salesagility/SuiteCRM#4085]
#### Issue

I had hoped updating to latest version of suiteCRM would fix. However, no joy. Presently running 7.9.4.

When I try to list tasks I'm presented with an error. With Ajax turned off for the module I'm presented with an empty page.

See here for screenshot with Javascript Console enabled to view the errors being thrown by SuiteCRM:
www.screencast.com/t/LCzVJv4BTfc

Additionally, the ability to close individual tasks from the home page is broken.

Please see:
www.screencast.com/t/juTil2ox79H AND
www.screencast.com/t/tIJJH6IHD

Which show the error messages that crop up as a result of the requested action.

#### Context
High priority as this impacts core functionality and makes globally addressing tasks impossible.

#### Your Environment

* SuiteCRM Version used: 7.9.4
* Browser name and version: Tested in Chrome Version 59.0.3071.115 (Official Build) (64-bit) and Firefox 54.0.1 (64-bit) Both on Mac OSX 10.12.6 
* Environment name and version (e.g. MySQL, PHP 7): MariaDB 10.0.31-MariaDB - MariaDB Server, PHP 5.6
* Operating System and version (e.g Ubuntu 16.04): CentOS 7.2 x64

**Comment by Dillon-Brown** [comment]
@gdginc I'm unable to replicate on 7.9.4. Have you tried clearing cache + setting correct permissions + repair? I would also check to see if you can replicate this on the demo: https://suitecrm.com/demo

**Comment by gdginc** [comment]
@Dillon-Brown Naturally I am unable to replicate on your demo space.

I have cleared caches. I have tried in numerous browsers and in incognito/private modes. I have run repair on each available option in the repair list. As I've just updated to 7.9.4, the upgrade wizard confirmed the permissions were all correct. Ran through all of the above before reaching out to you folks. Additionally, as best I can tell, we are running a compatible system. 

Is there any sort of "test" script available to verify the system meets & exceeds the requirements? Otherwise, what's the next step?
